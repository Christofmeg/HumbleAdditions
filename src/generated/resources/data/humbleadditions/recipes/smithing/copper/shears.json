{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:copper_ingot"
  },
  "base": {
    "item": "humbleadditions:stone_shears"
  },
  "result": {
    "item": "humbleadditions:copper_shears"
  }
}