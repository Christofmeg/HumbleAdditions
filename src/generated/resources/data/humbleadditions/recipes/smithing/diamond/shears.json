{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:diamond_block"
  },
  "base": {
    "item": "minecraft:shears"
  },
  "result": {
    "item": "humbleadditions:diamond_shears"
  }
}