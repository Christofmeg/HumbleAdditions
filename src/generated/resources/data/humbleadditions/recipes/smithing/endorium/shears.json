{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:endorium_block"
  },
  "base": {
    "item": "humbleadditions:diamond_shears"
  },
  "result": {
    "item": "humbleadditions:endorium_shears"
  }
}