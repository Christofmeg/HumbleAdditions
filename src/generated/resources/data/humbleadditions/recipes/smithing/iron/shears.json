{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:iron_ingot"
  },
  "base": {
    "item": "humbleadditions:copper_shears"
  },
  "result": {
    "item": "minecraft:shears"
  }
}