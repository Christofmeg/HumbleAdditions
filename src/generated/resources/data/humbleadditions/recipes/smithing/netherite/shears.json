{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:netherite_block"
  },
  "base": {
    "item": "humbleadditions:diamond_shears"
  },
  "result": {
    "item": "humbleadditions:netherite_shears"
  }
}