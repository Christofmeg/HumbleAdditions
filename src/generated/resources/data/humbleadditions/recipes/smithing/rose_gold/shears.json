{
  "type": "minecraft:smithing",
  "addition": {
    "item": "humbleadditions:rose_gold_ingot"
  },
  "base": {
    "item": "humbleadditions:golden_shears"
  },
  "result": {
    "item": "humbleadditions:rose_gold_shears"
  }
}