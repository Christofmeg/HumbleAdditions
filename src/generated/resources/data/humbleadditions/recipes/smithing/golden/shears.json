{
  "type": "minecraft:smithing",
  "addition": {
    "item": "minecraft:gold_ingot"
  },
  "base": {
    "item": "humbleadditions:wooden_shears"
  },
  "result": {
    "item": "humbleadditions:golden_shears"
  }
}